public/javascripts/controllers.js

```diff
@@ -888,82 +857,50 @@ angular.module('uforia')
             $scope.cuErrorMessages = [];
-
-            // Checks
-            if(typeof user.firstName === "undefined"){
-              $scope.cuErrorMessages.push('First name is required');
-            }
-
-            if(typeof user.lastName === "undefined"){
-              $scope.cuErrorMessages.push('Last name is required');
-            }
-
-            if(typeof user.email === "undefined"){
-              $scope.cuErrorMessages.push('Email is required');
-            }
-
-            if(typeof user.password === "undefined"){
-              $scope.cuErrorMessages.push('Password is invalid. Make sure the password is at least 4 characters long.');
-            }
-
-            if(typeof user.password2 === "undefined"){
-              $scope.cuErrorMessages.push('Make sure the passwords match.');
-            }
-
-            if($scope.cuErrorMessages.length === 0){
-
-              if(user.password !== user.password2){
-                $scope.cuErrorMessages.push('Password doesn\'t match.');
-
-              }else if(user.password === user.password2){
-                delete user.password2;
-                var addUser = true;
-
-                // Check if user exists
-                angular.forEach(users, function(value, key){
-                  var u = value._source;
-                  if(user.email == u.email){
-                    $scope.cuErrorMessages.push('An user with this email address already exists.');
-                    addUser = false;
-                  }
-                });
-
-                // Save user
-                if(addUser) {
-                  $http.post('/api/save_user', user)
-                      .success(function (data) {
-                            if (typeof data.error !== 'undefined') {
-                              $scope.error.push(data.error.message);
-                            }
-
-                            if (typeof data.response !== 'undefined') {
-                              if (data.response.created == true) {
-                                $scope.message.push('User has been added.');
-
-                                //Load new user in table
-                                $scope.rowCollection.push({
-                                  id: data.response._id, firstName: user.firstName, lastName: user.lastName, email: user.email,
-                                  role: 'N/A'
-                                });
-                                $scope.searchCollection = $scope.rowCollection;
-                              }
-                            }
-                            // Close modal
-                            $scope.modalInstance.dismiss();
-                          }
-                      );
-                }
-
-              }else{
-                $scope.cuErrorMessages.push('Unknown error.');
+            
+            var addUser = true;
+  
+            // Check if user exists
+            angular.forEach(users, function(value, key){
```

Comment:
Because we're considering in using MySQL for the users, we indeed postpone this to a later stadium.